src/components/App/App.tsx

```diff
@@ -75,6 +75,7 @@ interface AppState {
   currentSubtitle: string;
   currentMediaPaused: boolean;
   participants: User[];
+  subscribers: BooleanDict;
```

Comment:
isn't isSubscriber now a property on each user?

Reply:
oh I see, we don't have the roster in the chat component. In that case, having this is fine